src/showcase/nz-demo-button/nz-demo-button-loading.component.ts

```diff
@@ -18,30 +18,9 @@ import { Component, OnInit } from '@angular/core';
       <span>Click me!</span>
     </button>
     <br>
-    <button nz-button [nzLoading]="true" [nzShape]="'circle'"></button>
-    <button nz-button [nzLoading]="true" [nzType]="'primary'" [nzShape]="'circle'"></button>
+    <button nz-button nzLoading [nzShape]="'circle'"></button>
+    <button nz-button nzLoading [nzType]="'primary'" [nzShape]="'circle'"></button>
   `,
   styles  : []
 })
-export class NzDemoButtonLoadingComponent implements OnInit {
-  isLoadingOne = false;
-  isLoadingTwo = false;
-  loadOne = (value) => {
-    this.isLoadingOne = true;
-    setTimeout(_ => {
-      this.isLoadingOne = false;
-    }, 5000);
-  };
-  loadTwo = (value) => {
```

Comment:
will cause error when click `Click me!` button

Reply:
will check it.

Reply:
Thx, it's just by mistake.